.github/workflows/tests.yml

```diff
@@ -33,7 +33,8 @@ jobs:
           export CRIPT_TOKEN="125433546"
           export CRIPT_HOST="http://development.api.mycriptapp.org/"
           python3 -m pip install pytest
-          python3 -m pip install -r requirements.txt
+          python3 -m pip install -r requirements.txt 
+          python3 -m pip install -r requirements_dev.txt 
```

Comment:
we could just do `requirements_dev.txt` because that by default installs `requirements.txt`, but I think it's fine either way